Tests/MessagingInApp/MessagingInAppTest.swift

```diff
@@ -92,17 +90,12 @@ class MessagingInAppTest: UnitTest {
 extension MessagingInAppTest {
     private func assertModuleInitialized(isInitialized: Bool, givenEventListener: InAppEventListener?, file: StaticString = #file, line: UInt = #line) {
         if isInitialized {
-            XCTAssertEqual(hooksMock.addCallsCount, 1, file: file, line: line)
```

Comment:
hooks are no longer used in [MessagingInApp](https://github.com/customerio/customerio-ios/blob/main-replica-for-cdp/Sources/MessagingInApp/MessagingInApp.swift) class.